### PR TITLE
pupnp: update to 14.5

### PR DESCRIPTION
**Pull request by neheb**
Dereference and delete backport patches.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by neheb** [comment]
That's a strange error...

**Comment by neheb** [comment]
Oh I see the issue. Someone smart decided to use github's unstable tarballs for some reason.

**Comment by neheb** [comment]
Switched to using the normal tarball.

File: mingw-w64-pupnp/PKGBUILD [hunk 1]
```diff
@@ -37,22 +31,17 @@ apply_patch_with_msg() {
...
-  cd "${srcdir}/${_realname}-release-${pkgver}"
+  cd "${srcdir}/libupnp-${pkgver}"
   apply_patch_with_msg \
     002-libupnp-win32.patch \
-    005-backport-Fix-Windows-support-when-building-with-autotools.patch \
-    006-backport-mediatomb-fix-but-place-it-differently.patch \
-    007-backport-Fix-Windows-support-when-building-with-MinGW.patch \
-    008-backport-Use-int64_t-instead-of-long-long.patch
-
-  autoreconf -fi
```
Comment (lazka) [inline mingw-w64-pupnp/PKGBUILD:48]: I think autoreconf  is needed for clang support.

Comment (neheb) [reply mingw-w64-pupnp/PKGBUILD:48]: Oh unfortunate.

**Comment by neheb** [comment]
```
2021-05-02T21:08:32.5256773Z error: target not found: mingw-w64-ucrt-x86_64-cyrus-sasl
2021-05-02T21:08:32.5977903Z ==> ERROR: 'pacman' failed to install missing dependencies.
2021-05-02T21:08:32.6153038Z ==> Missing dependencies:
2021-05-02T21:08:32.6154141Z   -> mingw-w64-ucrt-x86_64-glew
2021-05-02T21:08:32.6156129Z   -> mingw-w64-ucrt-x86_64-cyrus-sasl
2021-05-02T21:08:33.2872415Z ==> ERROR: Could not resolve all dependencies.
2021-05-02T21:08:33.3903528Z ==> Removing installed dependencies...
```

comes as a result of the vlc bump. pupnp itself compiles file on UCRT64.

**Comment by lazka** [comment]
> comes as a result of the vlc bump. pupnp itself compiles file on UCRT64.

yes, this is fine, vlc doesn't build there yet, please ignore.

File: mingw-w64-pupnp/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-pupnp/PKGBUILD:48]: clang worked out now, thanks.

**Comment by lazka** [comment]
Thanks!

**Pull request merged**